src/components/formio/radio.tsx

```diff
@@ -51,11 +54,36 @@ export const Radio: React.FC<RadioProps> = ({
   options,
   label,
   required = false,
+  isPreview = false,
   tooltip = '',
   description = '',
 }) => {
+  const {values, setValues} = useFormikContext<ExtendedComponentSchema>();
```

Comment:
```suggestion
  const {values, setFieldValue} = useFormikContext<ExtendedComponentSchema>();
```

and then `onClick={() => setFieldValue(name, '')}`